x/builder/rewards_address_provider/fixed_address_rewards_address_provider.go

```diff
@@ -0,0 +1,46 @@
+package rewardsaddressprovider
+
+import (
+	"cosmossdk.io/depinject"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/skip-mev/pob/x/builder/types"
+)
+
+// Provides auction profits to a fixed address
+type FixedAddressRewardsAddressProvider struct {
```

Comment:
nit: sdk documentation convention is prefaced w/ struct / function / interface / etc. name, and then a brief description abt. what the thing does, expectations, failure cases, etc.